src/io.c

```diff
@@ -16,6 +16,7 @@ void io_seproxyhal_display(const bagl_element_t *element) {
 }
 
 uint8_t io_event(uint8_t channel) {
+    (void) channel;
```

Comment:
This is only to suppress a warning during build that this variable is not used, it does nothing.